src/app/pages/users/user/stripe/payment-methods/payment-methods.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+import { PaymentMethodsTableDataSource } from './payment-methods-table-data-source';
+
+@Component({
+  selector: 'app-payment-methods',
+  templateUrl: 'payment-methods.component.html',
```

Comment:
You may also add the 'providers' property like other components using data sources.